TOOLS/ce/src/main/scala/org.opalj.ce/HTMLExporter.scala

```diff
@@ -0,0 +1,58 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import java.io.File
+import java.io.PrintWriter
+import java.nio.file.Path
+import scala.io.Source
+import scala.util.Using
+
+import com.typesafe.config.Config
+
+/**
+ * Exports the Config structure into an HTML file.
+ * @param ConfigList Accepts a List of parsed Configuration Files.
+ * @param templatePath Accepts a Path to the HTML Template that should be used.
+ */
+class HTMLExporter(ConfigList: Iterable[ConfigNode], templatePath: Path) {
```

Comment:
Make this an object, moving `ConfigList` (with lower-case name) and `templatePath` to parameters of `exportHTML`